pro/src/new_components/VenueForm/EACInformation/__specs__/EACInformation.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import { createBrowserHistory } from 'history'
+import React from 'react'
+import { Provider } from 'react-redux'
+import { Router } from 'react-router-dom'
+
+import { IVenue } from 'core/Venue'
+import { configureTestStore } from 'store/testUtils'
+
+import EACInformation from '../EACInformation'
+
+const CREATION_TEXT = /Renseigner mes informations/
+const UPDATE_TEXT = /Modifier mes informations/
+const INFORMATION_BANNER_TEXT = /Une fois votre lieu créé/
+
+const renderEACInformation = async ({
+  venue,
+  isCreatingVenue,
+}: {
+  venue?: IVenue | null
+  isCreatingVenue: boolean
+}) => {
+  render(
+    <Router history={createBrowserHistory()}>
+      <Provider store={configureTestStore({})}>
+        <EACInformation isCreatingVenue={isCreatingVenue} venue={venue} />
+      </Provider>
+    </Router>
+  )
+}
+
+describe('components | EACInformation', () => {
+  it('should have the button disabled', async () => {
```

Comment:
On essaie d'écrire les tests du point de vue utilisateur, avec le use case:
quelque chose comme `should not be able to submit before filling the values`